.env

```diff
@@ -1 +1 @@
-PORT=3000
\ No newline at end of file
+FOO=BAR
```

Comment:
That's an example for people who want to use the `.env` file, but might be better simply mention it in the readme.